### PR TITLE
[USH-873] & [USH-890] Changed Text From Delete 1 posts to Delete 1 Post.

### PR DESCRIPTION
**Pull request by MugoBrian**
This PR fixes this two issues since they are related and affected the same area:
1. [Say "....deleted 1 post..." instead of "...deleted 1 posts..." after deleting 1 post](https://linear.app/ushahidi/issue/USH-873/say-deleted-1-post-instead-of-deleted-1-posts-after-deleting-1-post).
2. ["OK, Sounds good" text.](https://linear.app/ushahidi/issue/USH-890/ok-sounds-good-text)

Changes made:
1. Changed the grammar in the deleted 1 posts confirm modal description affected in the `feed.component.ts` and `post-head.component.ts`.
2. Edited the `en.json` accordingly.
3. Centralized "Success" text alignment.

### CODE REVIEW

**Comment by MugoBrian** [comment]
Hello @AmTryingMyBest and @tuxpiper, this PR is ready for review.

**Comment by tuxpiper** [comment]
Hi @MugoBrian . Sorry, I'm not sure I can review this one properly. @Ifycode could you please take a look?

File: apps/web-mzima-client/src/app/feed/feed.component.ts [hunk 1]
```diff
@@ -481,11 +481,12 @@ export class FeedComponent extends MainViewComponent implements OnInit {
     }
     this.selectedPosts = [];
     if (count) {
+      const postWord = count === 1 ? `post` : `posts`;
```
Comment (ushahidlee) [inline apps/web-mzima-client/src/app/feed/feed.component.ts]: Please change from backticks to quotes

**Review by ushahidlee — CHANGES_REQUESTED** [review]
Fine except for the quote characters

File: apps/web-mzima-client/src/app/feed/feed.component.ts (hunk 1 above)
Comment (MugoBrian) [reply apps/web-mzima-client/src/app/feed/feed.component.ts]: Ok. On it.

**Comment by MugoBrian** [comment]
Hello @ushahidlee. I have made the necessary changes.

**Review by ushahidlee — APPROVED** [review]
Great, thanks @MugoBrian !

**Pull request merged**